### PR TITLE
Somewhat romanian translations

### PR DESCRIPTION
**Pull request by andriejs**
Some Romanian Translations

### CODE REVIEW

File: public/locales/ro/about.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     "paragraph2": "Pentru a vă alătura serverului nostru, faceți clic pe următorul link: <0>https://discord.gg/g3Qnwy4V6A</0>. Abia așteptăm să ne vedem acolo!"
+   },
+   "privacyStatement": {
+     "title": "Declarație de confidențialitate",
+     "paragraph1": "Apreciem foarte mult confidențialitatea dumneavoastră și ne angajăm să protejăm confidențialitatea utilizatorilor noștri. Nu colectăm și nu stocăm niciun text pe care îl introduceți sau primiți de la serverul OpenAI sub nicio formă. Codul nostru sursă este disponibil pentru inspecția dvs. pentru verificați această afirmație.",
+     "paragraph2": "Prioritatezăm securitatea cheii dvs. API și o gestionăm cu cea mai mare atenție. Dacă utilizați propria cheie API, cheia dvs. este stocată exclusiv în browser și nu este niciodată partajată cu nicio entitate terță parte. Este folosită exclusiv pentru scopul propus de a accesa API-ul OpenAI și nu pentru orice altă utilizare neautorizată."
+   }
+}
```
Comment (ayaka14732) [inline public/locales/ro/about.json:27]: Should add a new line at the end of the file

File: src/constants/language.ts [hunk 2]
```diff
@@ -1,67 +1,71 @@
...
+	'ms',
+	'nb',
+	'sv',
+	// 'ug',
+	'yue',
+	'zh',
+	'zh-CN',
+	'zh-HK',
+	'zh-TW',
+	'ro',
```
Comment (ayaka14732) [inline src/constants/language.ts:22]: Wrong formatting

**Review by ayaka14732 — CHANGES_REQUESTED** [review]
Thank you for contributing the Romanian translation! It seems that there are some formatting issues. Besides, the PR title should be something like "Add Romanian translation".

File: src/constants/language.ts (hunk 2 above)
Comment (blindful) [reply src/constants/language.ts:22]: Sorry for the part with the wrong pr title.

**Review by ayaka14732 — APPROVED** [review]
Thank you!

**Review by ztjhz — APPROVED** [review]

**Pull request merged**